mail_activity_board/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2018 David Juaneda - <djuaneda@sdi.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Activities board",
```

Comment:
maybe we could apply the module name here
```suggestion
    "name": "Mail Activity Board",
```